core/templates/dev/head/filters.js

```diff
@@ -198,6 +202,10 @@ oppia.filter('parameterizeRuleDescription', [
             // The following case is for MultipleChoiceInput
             for (var i = 0; i < choices.length; i++) {
               if (choices[i].val === inputs[varName]) {
+                if (choices[i].label.indexOf('oppia-noninteractive') !== -1) {
+                  choices[i].label =
```

Comment:
Don't modify the input object. As a general rule, this is unsafe. You don't know what else is using it.

Reply:
Okay, will create a separate variable for it.